packages/components/src/markdown/index.stories.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { text, withKnobs } from '@storybook/addon-knobs';
+import { MemoryRouter } from 'react-router';
+import Markdown from './index';
+import longText from './long_text.fixture';
+
+const STORY_PARAMETERS = {
+  knobs: { escapeHTML: false },
+};
+
+export default { title: 'Markdown', component: Markdown, decorators: [withKnobs] };
+
+export const Default = () => (
+  <MemoryRouter>
```

Comment:
i propose to wrap every story in memory router on storybook level

Reply:
I propose not to do it. Stories act as usage examples and having the memory router in there shows the dependency to it

Reply:
fair point